tools/debian/convert_debian.py

```diff
@@ -265,9 +266,13 @@ def parse_webwml_files(advisories: Advisories, webwml_repo_path: str,
 
     advisory.references.append(Reference('ADVISORY', advisory_url))
 
-    git_relative_path = os.path.relpath(data_path, webwml_repo_path)
-    git_relative_data_paths[git_relative_path] = dsa_id
+    git_relative_path_wml = os.path.relpath(wml_path, webwml_repo_path)
+    git_relative_path_data = os.path.relpath(data_path, webwml_repo_path)
+    git_relative_paths[git_relative_path_wml] = dsa_id
+    git_relative_paths[git_relative_path_data] = dsa_id
 
+  modified_date_dict = collections.defaultdict(
+    lambda: datetime.fromtimestamp(0, timezone.utc))
```

Comment:
Is this indented correctly? Typically these line continuations would need double (i.e. 4 spaces) here.

Reply:
Ah, nice catch, ran pyformat now. I think I should to put pyformat on a pre commit hook to have it run automatically